.vsts-dotnet-ci.yml

```diff
@@ -4,8 +4,11 @@
 
 resources:
   containers:
-  - container: LinuxContainer
+  - container: LinuxCentosContainer
```

Comment:
I don't think these need the prefix `Linux`.  `CentosContainer` and `UbuntuContainer` is enough IMO.